storm/src/main/storm/mesos/resources/ResourceEntries.java

```diff
@@ -93,15 +93,15 @@ public ReservationType getReservationType() {
     public RangeResourceEntry add(ResourceEntry<Long> resourceEntry) {
```

Comment:
i.e., please try to answer these "asks" from @JessicaLHartog : 

> Before we continue this discussion, can you please provide some comments and/or an example for how this code is supposed to behave?
> 
> It would probably be most beneficial for my understanding if you provide this explanation without looking at the code aside from the signature provided here:
>    `public RangeResourceEntry add(RangeResourceEntry<Long> resourceEntry)`